### ISSUE TITLE
Project dependencies needs to be updated

### ISSUE THREAD

**Issue by benthemobileguy** [loadsmileau/native_updater#41]
The project breaks on newer flutter versions. I did a PR to fix that.

**Comment by daniJimen** [comment]
@benthemobileguy you are right, i think that library could be a good tool to manage updates in app it might be worth updating even today. I was preparing a PR to update it completely, please give me your support to try and get them to update
PR: https://github.com/ofload/native_updater/pull/42

**Comment by milhomem** [comment]
done, prs merged https://pub.dev/packages/native_updater
